common/browser_context.go

```diff
@@ -102,6 +109,26 @@ func (b *BrowserContext) AddInitScript(script goja.Value, arg goja.Value) error
 	return nil
 }
 
+func (b *BrowserContext) parseObjectInitScript(script goja.Value) (string, error) {
+	obj := script.ToObject(b.vu.Runtime())
+	for _, k := range obj.Keys() {
+		switch k {
+		case "content":
+			return obj.Get(k).String(), nil
+		case "path": //nolint:goconst
+			path := obj.Get(k).String()
+			source, err := readFile(path)
+			if err != nil {
+				return "", fmt.Errorf("can't read script from path '%s': %w", path, err)
```

Comment:
The error stutters: `parsing init script: can't read script from path 'static/dinit_script.js': open /Users/inanc/grafana/k6browser/main/tests/static/dinit_script.js: no such file or directory`.

We can make it concise and informative at the same time:

```suggestion
				return "", fmt.Errorf("reading file: %v", err)
```

This will result in the following:

`parsing init script: reading file: open /Users/inanc/grafana/k6browser/main/tests/static/dinit_script.js: no such file or directory`